src/pages/Pokemon/Evolution.js

```diff
@@ -1,10 +1,36 @@
 import React from "react";
-import Image from "./Image";
+import { useNavigate } from "react-router-dom";
+import { formatName, formatNumber } from "../../utils";
+import Type from "../../components/Type/Type";
+import "./pokemon.css";
 
-const Evolution = () => {
+const Evolution = ({ pokemonItem }) => {
+  let navigate = useNavigate();
+  const evolutions = pokemonItem?.evoListItems?.map((item) => (
+    <div className="GIJOE" onClick={() => navigate(`/${item.name}`)}>
```

Comment:
mate yeah we probs wanna change these classNames now so we don't forget later on 😂